chef/cookbooks/opendaylight/attributes/default.rb

```diff
@@ -17,4 +17,4 @@
 # limitations under the License.
 #
 
-default[:opendaylight][:logfile] = "/dev/null"
+default[:opendaylight][:features] = "odl-restconf odl-l2switch-switch odl-dlux-core odl-ovsdb-openstack"
```

Comment:
It will look good if we do that but it will cause confusion. There are around  20-30 features in ODL and as we not doing anything additional other than installing them, we will have to include all of them  in subsections. For now this is ok i think. Later once we start fully supporting ODL features, we can add check box for each feature.